requirements.txt

```diff
@@ -23,51 +22,12 @@ typed-ast==1.4.0
 wcwidth==0.1.7
 wrapt==1.11.2
 zipp==0.6.0
-astroid==2.2.5
-atomicwrites==1.3.0
-attrs==19.1.0
-autopep8==1.4.4
-certifi==2019.6.16
-Click==7.0
-importlib-metadata==0.20
-isort==4.3.21
-lazy-object-proxy==1.4.2
-mccabe==0.6.1
-more-itertools==7.2.0
-packaging==19.1
-pluggy==0.12.0
-py==1.8.0
-pycodestyle==2.5.0
-pylint==2.3.1
-pyparsing==2.4.2
-pytest==5.1.2
-six==1.12.0
-typed-ast==1.4.0
-wcwidth==0.1.7
-wrapt==1.11.2
-zipp==0.6.0
-astroid==2.2.5
-atomicwrites==1.3.0
-attrs==19.1.0
-autopep8==1.4.4
-certifi==2019.6.16
-Click==7.0
-importlib-metadata==0.20
-isort==4.3.21
-lazy-object-proxy==1.4.2
-mccabe==0.6.1
-more-itertools==7.2.0
-packaging==19.1
-pluggy==0.12.0
-py==1.8.0
-pycodestyle==2.5.0
 pydocstyle==4.0.1
-pylint==2.3.1
-pyparsing==2.4.2
-pytest==5.1.2
-six==1.12.0
 snowballstemmer==1.9.0
-typed-ast==1.4.0
-wcwidth==0.1.7
-wrapt==1.11.2
-zipp==0.6.0
+beautifulsoup4==4.8.2
+bs4==0.0.1
+chromedriver-binary==81.0.4044.20.0
```

Comment:
May want to change to `chromedriver-binary==79.0.3945.16.0` or some 80 versions.
81 is still in beta now🤣🤣